tests/unit/disco/components/TestAddon.js

```diff
@@ -443,42 +444,103 @@ describe(__filename, () => {
   describe('addon with type static theme', () => {
     const fullImage = 'https://addons.cdn.mozilla.net/full/54321.png';
     const newAddonName = 'Summertime';
+    let install = sinon.spy();
+    let enable = sinon.spy();
 
-    it('renders the correct image', () => {
-      const shallowRoot = renderAddon({
+    const renderWithStaticTheme = (props = {}) => {
+      return renderAddon({
         addon: {
           ...result,
           name: newAddonName,
+          type: ADDON_TYPE_STATIC_THEME,
           previews: [
             {
               ...fakePreview,
               image_url: fullImage,
             },
           ],
-          type: ADDON_TYPE_STATIC_THEME,
         },
+        status: UNINSTALLED,
+        install,
+        enable,
+        hasAddonManager: true,
+        getAddonEnabledStatus: sinon.stub().returns(false),
+        ...props,
       });
-      const image = shallowRoot.find('.Addon-theme-header-image');
+    };
+
+    it('renders the correct image', () => {
+      const root = renderWithStaticTheme();
+      const image = root.find('.Addon-theme-header-image');
       expect(image).toHaveProp('src', fullImage);
       expect(image).toHaveProp('alt', `Preview of ${newAddonName}`);
     });
+
+    it("calls install and enable helper functions when clicking on the static theme's header image if hasAddonManager is true", async () => {
```

Comment:
I am wondering if it's better to group some of these tests here in a `hasAddonManager is false` (or true) describe?